usecases/config/config_handler.go

```diff
@@ -103,6 +103,7 @@ type Config struct {
 	DisableGraphQL                      bool                     `json:"disable_graphql" yaml:"disable_graphql"`
 	AvoidMmap                           bool                     `json:"avoid_mmap" yaml:"avoid_mmap"`
 	CORS                                CORS                     `json:"cors" yaml:"cors"`
+	Telemetry                           bool                     `json:"telemetry" yaml:"telemetry"`
```

Comment:
given the nature that it gonna be enabled by default what do you think if we make it `disable_telemetry` ?

Reply:
While I see your point @daveatweaviate, there is already a pattern of naming according to the default, such as `AvoidMmap`, `DisableGraphQL`, `DisableLazyLoadShards`, etc.

I'll go with `DisableTelemetry`

Reply:
Please note @daveatweaviate that the env var has been renamed to `DISABLE_TELEMETRY` as well